src/strategies/base.ts

```diff
@@ -69,6 +69,8 @@ export interface BaseStrategyOptions {
   includeComponentInTag?: boolean;
   pullRequestTitlePattern?: string;
   extraFiles?: ExtraFile[];
+  // Java-only
+  snapshotLabels?: string[];
```

Comment:
Can we move this to a `JavaStrategyOptions` similar to Ruby: https://github.com/googleapis/release-please/blob/d3c5be564e4b1034e35d0b4d87d0337b0370772c/src/strategies/ruby.ts#L26-L28

Reply:
Done